IntegrationAdapters/Controllers/MedicineController.cs

```diff
@@ -115,6 +115,11 @@ public IActionResult GetActionsAndBenefits()
             return Ok(Program.ResponseMessageGrpc);
         }
 
-        
-    }
-}
+        [HttpGet("getMedicine")]
+        public IActionResult GetMedicine()
+        {
+            medicineService.AddMedicine();

```

Comment:
Is this a mistake?

Reply:
Yes, it is mistake. It was add only for adding data in our local database. We will delete it. Thanks

Reply:
No problem 😄